omod/pom.xml

```diff
@@ -196,4 +215,64 @@
 		</plugins>
 	</build>
 
+	<!-- different profiles for building/testing against different version of 
+		openmrs -->
+ 
+	<profiles>
+		<profile>
+			<id>openmrs-1.9</id>
+
+			<properties>
+				<openMRSVersion>1.9.9</openMRSVersion>
+				<openMRSMinorVersion>1.9</openMRSMinorVersion>
+				<webservicesRestVersion>2.12</webservicesRestVersion>
+			</properties>
+			
+			<!-- <dependencyManagement> -->
+				<dependencies>	
+					<dependency>
+						<groupId>org.openmrs.module</groupId>
+						<artifactId>webservices.rest-omod-common</artifactId>
+						<version>${webservicesRestVersion}</version>
+						<scope>provided</scope>
+					</dependency>
+				</dependencies>
+
+			<!-- </dependencyManagement> -->
+
+
+		</profile>
+		
+		<profile>
+			<id>openmrs-2.0</id>
+
+			<properties>
+				<openMRSVersion>2.0.0-SNAPSHOT</openMRSVersion>
+				<openMRSMinorVersion>2.0</openMRSMinorVersion>
+				<legacyUIVersion>1.0-SNAPSHOT</legacyUIVersion>
+				<webservicesRestVersion>2.13-SNAPSHOT</webservicesRestVersion>
+			</properties>
+			
+			<!-- <dependencyManagement> -->
+				<dependencies>	
+					<dependency>
+						<groupId>org.openmrs.module</groupId>
+						<artifactId>legacyui-omod</artifactId>
+						<version>${legacyUIVersion}</version>
+						<type>jar</type>
+						<scope>provided</scope>
+					</dependency>
+					<dependency>
+						<groupId>org.openmrs.module</groupId>
```

Comment:
Seeme like this is a common dependency with the same version and scope, why not take it out of the profiles?